upstream-community-operators/jaeger/jaeger.v1.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: jaeger-operator.v1.11.0
+  replaces: jaeger-operator.v1.8.2
+  namespace: placeholder
+  annotations:
+    categories: tracing, monitoring, troubleshooting
```

Comment:
Please use categories outlined in our docs here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories